apps/vr-tests-react-components/src/stories/Dialog/Dialog.stories.tsx

```diff
@@ -442,3 +442,59 @@ ScrollLongContent.storyName = 'scroll long content';
 export const ScrollLongContentDarkMode = getStoryVariant(ScrollLongContent, DARK_MODE);
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27229&commitId=cc3ceb0f2016d31c1406561fd0b6362f11d4a720&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1901 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Fluid Actions End.chromium.png|0| Added|
|Dialog.Fluid Actions Start.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>